common/oauthTokenManager.go

```diff
@@ -731,6 +731,9 @@ func jsonToTokenInfo(b []byte) (*OAuthTokenInfo, error) {
 	if err := json.Unmarshal(b, &OAuthTokenInfo); err != nil {
 		return nil, err
 	}
+	if OAuthTokenInfo.TokenRefreshSource == TokenRefreshSourceTokenStore {
+		_, _ = OAuthTokenInfo.GetTokenStoreCredential()
```

Comment:
So there are other values that user can have for refresh source here. I see only in case of token store we are going for refresh and not in other cases, so where do we refresh the token for other cases.